app/views/layouts/application.html.erb

```diff
@@ -11,9 +11,15 @@
   </head>
 
   <body>
-    <%= yield %>
-    <% if flash[:alert].present? %>
-        <p class="my_alert"><%= flash[:alert] %></p>
-    <% end %>
+    <div class="container-fluid">
+      <div class="container">
+        <%= render "layouts/navbar"%>
+        <%= render "layouts/hero"%>
+        <%= yield %>
+        <% if flash[:alert].present? %>
+            <p class="my_alert"><%= flash[:alert] %></p>
```

Comment:
The lesson unfortunately didn't show how you're supposed to use Bootstrap alerts, so I'll show the code:

```suggestion
            <p class="alert alert-danger"><%= flash[:alert] %></p>
```

The `flash[:notice]` container is also supposed to be here instead of duplicated across several of the views below:

```erb
<% if flash[:notice].present? %>
  <div class="alert alert-success"><%= flash[:notice] %></div>
<% end %>
```

These are taken from the Bootstrap docs: <https://getbootstrap.com/docs/5.3/components/alerts/>

Reply:
ok, got it